auth/internal/httpapi/router.go

```diff
@@ -0,0 +1,24 @@
+package httpapi
+
+import (
+	"auth/internal/httpapi/handlers"
+	"lib/auth"
+	"lib/ginhelper"
+)
+
+func (a *HTTPAPI) regiesterRoutes() {
+	tokenHandlers := handlers.NewTokens(a.authService)
+	userHandlers := handlers.NewUsers(a.userService)
+
+	a.router.Use(ginhelper.ErrorMiddleware())
+
+	authorized := a.router.Group("/")
```

Comment:
sometimes it is convenient to use specific prefixes for each internal service '/fb-auth' (fb - family budget) - it will help to not mixed up it with routes from other services - simplify overall system routing.